packages/loader/test/index.test.js

```diff
@@ -1,19 +1,16 @@
 /* @jsx React.createElement */
 /* @jsxFrag React.Fragment */
-const path = require('path')
-const webpack = require('webpack')
-const MemoryFs = require('memory-fs')
-const React = require('react')
-const {renderToString} = require('react-dom/server')
-const _extends = require('@babel/runtime/helpers/extends')
-const _objectWithoutProperties = require('@babel/runtime/helpers/objectWithoutProperties')
-const {mdx} = require('../../react')
+import {fileURLToPath} from 'url'
+import fs from 'fs'
+import webpack from 'webpack'
+import React from 'react'
+import {renderToString} from 'react-dom/server'
+import remarkSlug from 'remark-slug'
 
 const transform = (filePath, options) => {
   return new Promise((resolve, reject) => {
-    // Webpack 5: const fs = new MemoryFs()
     const compiler = webpack({
-      context: __dirname,
+      context: fileURLToPath(new URL('.', import.meta.url)),
```

Comment:
[No `__filename` or `__dirname` in ESM](https://nodejs.org/api/esm.html#esm_no_filename_or_dirname).